slider/beatmap.py

```diff
@@ -1414,6 +1558,25 @@ def ar(self,
 
         return ar
 
+    @lazyval
+    def breaks(self):
+        """The breaks with all other events filtered out.
+        """
+        return tuple(e for e in self.events if not isinstance(e, Break))
```

Comment:
this seems to be the exact complementary set of events that you want (ie all non breaks)

Reply:
Oops yeah, copy pasta'd from the other stuff didn't notice the `not`.